pkg/virt-controller/watch/vmi.go

```diff
@@ -528,16 +528,6 @@ func (c *VMIController) updateStatus(vmi *virtv1.VirtualMachineInstance, pod *k8
 					break
 				}
 
-				hotplugPodsReady := false
-				hotplugPodsReady, syncErr = c.hotplugPodsReady(vmi, pod)
-				if syncErr != nil {
-					break
-				}
-				if !hotplugPodsReady {
-					log.Log.V(3).Object(vmi).Infof("Postpone the pod hand-over and await the attachment pod ready")
-					break
-				}
-
```

Comment:
So if I understood @jean-edouard correctly the virt-launcher pod is created, which then allows the attachment pod to get the selinux labels from the virt-launcher pod, but the virt-launcher pod waits for the attachment of the hotplugged volumes to be completed before starting the VM process. So that should give us the required guarantee.

Reply:
Currently virt-launcher no more waits for the attachment pod and the vm starts running right away.